src/modules/display.js

```diff
@@ -1,39 +1,129 @@
-const listContainer = document.querySelector('.todolist');
-
-const tasks = [
-  {
-    index: 0,
-    description: 'one',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'jump outta space',
-    completed: false,
-  },
-  {
-    index: 2,
-    description: 'watch fast x',
-    completed: false,
-  },
-  {
-    index: 3,
-    description: 'get some sleep',
-    completed: false,
-  },
-];
-
-const displayTask = (tasks) => {
-  const displaytTaskDem = tasks.map((item) => `
-  <div class="list" draggable="true">
-    <div class="content">
-        <input class="box" type="checkbox" name="" id="">
-        <input type="text" name="" id="" value="${item.description}">
-    </div>
-    <i class="fa-solid fa-ellipsis-vertical icon"></i>
-</div>`).join('');
-
-  listContainer.innerHTML = displaytTaskDem;
-};
-
-export default displayTask(tasks);
\ No newline at end of file
+export const InputField = document.querySelector('.addtodo');
+
+class Todo {
+  static todoLists = JSON.parse(localStorage.getItem('todoLists')) || [];
+
+  static inputField;
+
+  static index;
+
+  static completed
+
+  static listContainer;
+
+  static i = 0;
+
+  constructor(index, completed, description) {
```

Comment:
- [optional] The `todos'` status (**completed**) is missing from your project. I so much believe that you will implement this because you'll actually need it for the next milestone. 👍